src/dev-app/dev-app/dev-app-layout.ts

```diff
@@ -115,4 +116,18 @@ export class DevAppLayout {
       this._overlayContainer.getContainerElement().classList.remove(darkThemeClass);
     }
   }
+
+  toggleStrongFocus() {
+    const strongFocusClass = 'demo-strong-focus';
+
+    this.strongFocus = !this.strongFocus;
+
+    if (this.strongFocus) {
+      this._element.nativeElement.classList.add(strongFocusClass);
```

Comment:
Any reason not to just toggle the class on `document.body`?

Reply:
No reason, just essentially duplicated the logic in `toggleTheme`. I assumed there was some reason behind it. I can change them both to toggle the class on `document.body`.